### PR TITLE
remove phedexOnly flag

### PR DESCRIPTION
**Pull request by ticoann**
related to #6286

### CODE REVIEW

**Comment by ticoann** [comment]
@AndresTanasijczuk, Hi Andres, I remove the phedexOnly flag since now we have the behavior you want. Please review.

File: src/python/WMCore/Services/DBS/DBS3Reader.py [hunk 1]
```diff
@@ -613,7 +593,20 @@ def listFileBlockLocation(self, fileBlockNames, dbsOnly = False, phedexOnly = Fa
...
+                msg = "Error while getting block location from PhEDEx for block_name=%s)\n" % fileBlockNames
+                msg += "%s\n" % str(ex)
+                raise Exception(msg)
 
+            for name, nodes in blocksInfo.iteritems():
+                valid_nodes = set(nodes) - node_filter
+                if valid_nodes:  # dont add if only 'UNKNOWN' or None then get with dbs
+                    locations[name] = list(valid_nodes)
+
+            fileblockNames = set(fileBlockNames) - set(locations) #get the blocks we did not find information in phedex
```
Comment (AndresTanasijczuk) [inline src/python/WMCore/Services/DBS/DBS3Reader.py]: @ticoann You can remove this line as well.

**Comment by AndresTanasijczuk** [comment]
@ticoann Fine with me.

File: src/python/WMCore/Services/DBS/DBS3Reader.py (hunk 1 above)
Comment (ticoann) [inline src/python/WMCore/Services/DBS/DBS3Reader.py]: @AndresTanasijczuk ,Thanks Andres for catching this. Removed.

**Comment by ericvaandering** [comment]
test this please

**Comment by ericvaandering** [comment]
test this please

**Comment by ericvaandering** [comment]
test this please

**Pull request merged**